### ISSUE TITLE
Visual assertion about debug mode

### ISSUE THREAD

**Issue by slush0** [trezor/trezor-core#172]
Currently there's no mechanism to see device is in Debug mode (similar to trezor-mcu).

Implement visual verification and double protection against running Debug mode in production firmware.

**Comment by jpochyla** [comment]
Debug firmware has a red indicator in the corner.

**Comment by prusnak** [comment]
@jpochyla Correct, but we don't run display_refresh too often on hardware. Maybe we could set `loop.after_step_hook = display.refresh` in hardware debug build too?

https://github.com/trezor/trezor-core/blob/master/src/trezor/ui/__init__.py#L18

**Comment by jpochyla** [comment]
@slush0 Additional `__debug__` check implemented in 01bc12ec272f6050de03c17b1e98de2dbf0ce11b
